kazoo/protocol/connection.py

```diff
@@ -567,7 +575,10 @@ def _connect_attempt(self, host, port, retry):
                         response = self._read_socket(read_timeout)
                         close_connection = response == CLOSE_RESPONSE
                     else:
-                        self._send_request(read_timeout, connect_timeout)
+                        close_connection = self._send_request(
+                            read_timeout,
+                            connect_timeout,
+                        ) == SESSION_KEPT
```

Comment:
This feels a bit awkward, tacking on a boolean eval at the end. Can it be changed to get the response first as was done immediately above it?